bundles/org.platformio.eclipse.ide.home.ui/plugin.xml

```diff
@@ -108,8 +116,23 @@
                   label="%command.clean.name"
                   style="push">
             </command>
+            <command
+                  commandId="org.platformio.eclipse.ide.home.ui.terminal"
+                  icon="platform:/plugin/org.eclipse.ui.console/icons/full/cview16/console_view.png"
+                  label="%command.terminal.name"
+                  style="push">
+            </command>
          </toolbar>
       </menuContribution>
    </extension>
+   
+   <extension
+         point="org.eclipse.ui.console.consoleFactories">
+      <consoleFactory
+            class="org.platformio.eclipse.ide.home.internal.ui.terminal.TerminalFactory"
+            icon="platform:/plugin/org.eclipse.ui.console/icons/full/cview16/console_view.png"
+            label="%terminal.label">
```

Comment:
the key is missing one segment, I would suggest `consoleFactory.<id_of_your_factory>.label`